BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Antenna_Performance_Metric/rootAnalysis.py

```diff
@@ -1,297 +1,238 @@
 #!/usr/bin/env python
 #This file came from Will with some GENETIS specific modifications by Dylan
-import ROOT
-import numpy as np
-#import matplotlib.pyplot as plt
-#import scipy.signal
-from collections import defaultdict
+# Run with:
+# python rootAnalysis.py 1 1 19 /users/PAS1960/dylanwells1629/developing/GENETIS_PUEO/BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Generation_Data 2023_05_08 /users/PAS1960/dylanwells1629/developing/GENETIS_PUEO/BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop
+
+# Imports
 import os
-from fnmatch import fnmatch
-#import matplotlib
 import argparse
-parser = argparse.ArgumentParser()
+from collections import defaultdict
+from fnmatch import fnmatch
+
+import numpy as np
+
+import ROOT
+
+# Global Variables
+desc = 'Analyze the root files and output fitness scores and errors'
+parser = argparse.ArgumentParser(description = desc)
 parser.add_argument("gen", help="current gen", type=int)
 parser.add_argument("indiv", help="Individual", type=int)
 parser.add_argument("energy", help="Individual", type=int)
```

Comment:
Is this supposed to be `energy`?
```suggestion
parser.add_argument("energy", help="Energy", type=int)
```
A more verbose output might also be better.

Reply:
Yes, good catch! I added some more descriptive comments overall.